pyproject.toml

```diff
@@ -104,21 +104,17 @@ substitutions = [
 docs = [
     "autoflake",
     "mkdocs",
-    "mkdocs-embed-external-markdown",
     "mkdocs-exclude",
     "mkdocs-material",
     "mkdocs-redirects",
-    "mkdocs-simple-hooks",
     "mkdocstrings-python",
     "tomli",
     "pyupgrade",
-    "mike @ git+https://github.com/jimporter/mike.git@master",
-    "mkdocs-embed-external-markdown>=2.3.0",
+    "mike",
     "pytest-examples>=0.0.13",
     "pydantic-settings>=2.0b1",
     "pydantic-extra-types @ git+https://github.com/pydantic/pydantic-extra-types.git@main",
-    # Not available before v3.8.6, hence the pin here, as it's required by griffe
-    "backports-strenum==1.3.1; python_version >= '3.8.6' and python_version < '3.11'"
```

Comment:
This is one of the reasons I dislike the PDM behavior (as we discussed internally). We can end up with unused dependencies that have to be manually checked.

Reply:
Yeah, I wasted some time on this a while ago and it was relatively... annoying